app/frontend/src/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -97,6 +101,11 @@ export default function HostRequestSendField({
 
   const isButtonLoading = isLoading || isResponseLoading;
 
+  const isRequestClosed =
+    hostRequest.toDate < new Date().toISOString().split("T")[0] ||
```

Comment:
Feels like we are doing this same check on the backend as well... Couldn't we return this too as part of the requests object?

Reply:
Good idea

Reply:
Actually on second though, I want to just leave this as is, because we are getting rid of host request messages later so I don't want to waste time on it (and we will just have to remove that field from the protobuf message again later)